wagtail_localize/tests/test_translations_report.py

```diff
@@ -135,6 +135,17 @@ def test_filter_by_source_locale(self):
         self.assertNotIn(self.blog_index_translation, response.context["object_list"])
         self.assertNotIn(self.blog_post_translation, response.context["object_list"])
 
+    def test_filter_by_source_locale_all_options(self):
```

Comment:
would be good to test that the locale filter gives you the `get_content_languages` choices

Reply:
Done